tests/hook_main.py

```diff
@@ -1044,7 +1050,10 @@ def discount_code_post(transaction):
     :param transaction:
     :return:
     """
-    transaction['skip'] = True
+    with stash['app'].app_context():
+        event = EventFactoryBasic()
```

Comment:
@magdalenesuo It will be event as for testing POST method, we don't need to create a discount code object, but we need an event object for the relationship.